짝수와-홀수.js

```diff
@@ -0,0 +1,5 @@
+function solution(num) {
+    var answer = "";
```

Comment:
변수 선언할 때 var대신 let, const를 사용해 주세요
```suggestion
    let answer = "";
```